AnalyticalGraphicsInc/cesium
# Replace jscoverage with karma-coverage

1. Replace our ancient, windows-only jscoverage implementation with karma-coverage, which works on all platforms and does not require a separate instrumentation step. You now run coverage by simply typing `npm run coverage`. This also provides improved results, such as branch coverage, which the old system did not.
2. Updated Testing Guide
3. Remove old instrument link from index.html
4. Remove jscoverage and related code.

__Current stats__
90.38% Statements 81603/90285
81.91% Branches 27629/33729
90.95% Functions 9010/9906
90.38% Lines 80611/89193

these numbers were lower than I expected, but `Source/Workers` is uncovered by unit tests (at least directly), we should probably add some tests for them and it would get back above 93% pretty easily. I didn't want to lie to ourselves by simple excluding them.

For anyone wondering why we stayed on the old system for so long, I actually made this change ~3 years ago but `karma-coverage` fixed a showstopping bug ~8 days ago and that finally allowed us to use it.
